### DISCUSSION TITLE
New W11 update (KB5023778) makes the File Explorer crash whenever it tries to show the Start Menu

### DISCUSSION THREAD

**Discussion by Tiagoquix** [valinet/ExplorerPatcher#1230] [General]
New update link: https://support.microsoft.com/en-us/topic/march-28-2023-kb5023778-os-build-22621-1485-preview-d490bb51-492e-410c-871f-50ad01b0f765

ExplorerPatcher and File Explorer works fine, except the Start Menu, which doesn't open and crashes when I try to open it.

---------

Strangely, I managed to fix the problem.

Just disabled the "Show Search button" option in the application properties and restarted explorer.exe a few times. After that, everything was back to normal.

In the new update, they changed Windows Search in some way, which made the Search button reappear on my taskbar (I don't use it). Maybe the Search button or the Windows Search update have something to do with the crash.

**Comment by Tiagoquix** [comment]
https://github.com/valinet/ExplorerPatcher/issues/1211

**Comment by ghost** [comment]
Can confirm same issue, and can confirm it being an issue with (KB5023778). 20 minutes ago I did the update to (KB5023778), before that it worked fine. After installation I am getting [This error](https://imgur.com/a/c7q5uXV) after a few crash loops.

Oddly what OP suggested with the toggling of show search button is not working for me. And it is disabled by default for me as well. Maybe it is contributing to the crash but I am not too sure that is the actual issue.

**Comment by ghost** [comment]
> Strangely, I managed to fix the problem.
> 
> Just disabled the "Show Search button" option in the application properties and restarted explorer.exe a few times. After that, everything was back to normal.

@Tiagoquix I'm curious what your start menu settings are, because I can't keep EP from crashing no matter what I do (it crashes when I try to open the start menu). My start menu is set to W10 style, center, fullscreen, and 'show more tiles'.

Interestingly, once EP crashes, I keep my W10 start menu, but the taskbar reverts to W11.

**Comment by ghost** [comment]
I actually did manage to figure out how to keep EP from crashing when opening the start menu.

Once I am signed into W11, I start the EP properties panel and go to Start Menu > Start Menu style > change it to Windows 11 > Restart File Explorer.

After it reloads, I change it back to Windows 10 style, and restart FIle Explorer once more. And then I will have W10 start menu as well as W10 taskbar and nothing crashes.

Edit: I've also noticed that after doing this once, EP no longer crashes, even on sign-in or system restart.

**Comment by jackyld** [comment]
I've tried all these suggestions and none of them help. It always crashes when I click the Start Menu after this update.

I kept messing around with settings and I think the only thing that worked for me was enabling the "Enable symbols download" in Advanced tab in ExplorerPatcher and restarting. After that I tried messing with the settings again and couldn't reproduce the crash.

**Reply by ghost** [reply]
The only thing I can think of that I didn't mention previously is that I uninstalled and reinstalled EP before I did any of the other things I mentioned above.

So all in all, my list of steps that I completed:

1. Uninstalled EP
2. Reinstalled EP
3. At this point I do remember that the app automatically updated "symbols".
4. Let EP crash three times and send error pop-up, dismissed it.
5. Searched for ExplorerPatcher in Windows search, and clicked on it.
6. ExplorerPatcher > Start Menu > Start Menu style > Windows 11 > Restart File Explorer.
7. ExplorerPatcher > Start Menu > Start Menu style > Windows 10 > Restart File Explorer.

**Reply by jackyld** [reply]
That's almost exactly what I did, a lot of re-installs/dismisses too. As soon as I enabled that symbol download I saw the popup of symbols downloaded at the bottom right after and then could no longer reproduce the crash. The symbols were required for a few things before and apparently now they're required altogether after the latest Windows update.

**Comment by mistertheloaf** [comment]
EP 22621.1413 worked fine.  Updated Windows to 22621.1485 and it completely Borked everything.  Fortunately I anticipated this and did a backup first.

Will have to play around with it some more.

**Comment by antoniu200** [comment]
EP needs to download symbols before some modifications work. Also see #1274, Start Menu with Recommended section disabled and Win-X menu on Windows 10 taskbar don't work without Symbols for the specific build of Windows you are using.

**Comment by DeanTroub** [comment]
Just installed 2023-04 Cumulative Update for Windows 11 Version 22H2 for x64-based Systems (KB5025239); and now ExplorerPatcher crashes whenever I try to open the start menu.

**Reply by mistertheloaf** [reply]
Starting with (KB5023778) that was released on March 28, Microsoft changed something in Explorer.exe 

Some people have reported that having ExplorerPatcher download new symbols  -- and a few reboots -- fixes the problem, but I haven't been able to get it to work yet.

**Comment by JakeWerecat** [comment]
Confirm the same issue after installing KB5025239. Start Button kept crashing explorer.exe.
Weird workaround suggested by Tiagoquix worked for me a well.
Another side effect for me is broken weather, identical issue like this https://github.com/valinet/ExplorerPatcher/discussions/1301

**Comment by Lovasz-Akos** [comment]
also crashing after update, but didn't even need to reinstall, just spam crashed it a few times until "new symbols downloaded" notification appeared, then it started working again
